aya/src/programs/cgroup_skb.rs

```diff
@@ -1,3 +1,4 @@
+//! CGroup SKB programs.
```

Comment:
I'd call these "Cgroup skb programs." "Cgroup sockaddr programs". "Cgroup sockopt programs." etc. That is, capital C just because it's the start of a sentence, then the well known thing like "skb" "sockaddr" etc with its usual spelling. And periods, periods everywhere!